doc/sphinx/proof-engine/ltac2.rst

```diff
@@ -464,14 +468,25 @@ Generic Syntax for Quotations
 In general, quotations can be introduced in terms using the following syntax, where
 :production:`quotentry` is some parsing entry.
 
-.. prodn::
-   ltac2_term += @ident : ( @quotentry )
 
 .. _ltac2_built-in-quotations:
 
 Built-in quotations
 +++++++++++++++++++
 
+.. insertprodn ltac2_quotations ltac1_expr_in_env
+
+.. prodn::
+   ltac2_quotations ::= ident : ( @lident )
+   | constr : ( @term )
+   | open_constr : ( @term )
+   | pattern : ( @cpattern )
+   | reference : ( {| & @ident | @qualid } )
```

Comment:
Is it feasible to remove this whitespace since the text says it's not permitted?
```suggestion
   | reference : ( {| &@ident | @qualid } )
```

Reply:
The grammar allows whitespace here, so I removed the text.

```
  globref:
    [ [ "&"; id = Prim.ident -> { CAst.make ~loc (QHypothesis id) }
      | qid = Prim.qualid -> { CAst.make ~loc @@ QReference qid }
    ] ]
  ;
```

Reply:
@ppedrot Can you comment on why this text was here? It's possible that the grammar is lying here.

Reply:
ping @ppedrot can you comment on the space in the grammar here? (submitting this as a comment review so that it shows up at the bottom of the PR and is not as easily missed)

Reply:
I'm not following the question anymore.

@Zimmi48 You're asking about the presence of "which parses either as :n:`@qualid` or :n:`&@ident`.", since removed??

> It's possible that the grammar is lying here.

Your evidence?

@JasonGross You're referring to the spaces here?  You think it should be just "ident:(" without spaces?  It's correct as it is.  Since the grammar is inserted automatically, eliminating the space after "ident" requires a a code change.  Not worth doing here.  Removing the space in ": (" is easier, but it doesn't seem worth it to me.

![image](https://user-images.githubusercontent.com/1253341/89838907-4ec29480-db21-11ea-8cd7-85ffb5160454.png)